### ISSUE TITLE
†Agnostidae not under †Agnostoidea

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#638]
https://www.catalogueoflife.org/data/taxon/64Y
https://www.catalogueoflife.org/data/taxon/622Y5

**Comment by yroskov** [comment]
Unfortunately, I don't know how technically to fix the issue.

**Comment by mdoering** [comment]
Indeed difficult if its not fixed in the source.
You could block the family from the sector and attach it manually under the superfamily

**Comment by yroskov** [comment]
Yeah, I am trying to follow your advice, but interface is slow in reflection of steps and show me confusing messages:

![image](https://github.com/CatalogueOfLife/data/assets/28574252/97c74143-be68-4823-9aea-479309c19770)

**Comment by yroskov** [comment]
...and I do not see Trilobita sector on the side of GSD (it is shown in CoL): 

![image](https://github.com/CatalogueOfLife/data/assets/28574252/8db847c1-439b-4c48-a8df-1a12c88bf58a)
https://www.checklistbank.org/catalogue/3/assembly?assemblyTaxonKey=7a77dbd5-bcff-42c9-b9fc-b05d5678302f&datasetKey=268676

All are very confusing now.

**Comment by yroskov** [comment]
>All are very confusing now.

I will be happy, if I succeed to return Trilobita sector as it was before my exercises.

**Comment by mdoering** [comment]
looks like all 3 are fine?
https://www.checklistbank.org/catalogue/3/sector?limit=100&offset=0&subjectDatasetKey=268676

Also in the assembly view:
<img width="1377" alt="image" src="https://github.com/CatalogueOfLife/data/assets/327505/1b45ba05-30b5-4f07-8970-8b920bbedab4">

I dont see the sector not found errors...

**Comment by mdoering** [comment]
Ah, but you cannot block the family and then use it as a new sector. The block decision is for the entire dataset source and will also block any new sector. Damn. Maybe we need an addition to the update decision and allow the selection of a new parent
